src/App.tsx

```diff
@@ -757,231 +760,94 @@ function App(props: RouteComponentProps) {
                 }}
             >
                 {!isMinimalMode && (
-                    <span
-                        style={{
-                            height: '50px',
-                            width: '100%',
-                            background: 'white',
-                            position: 'absolute',
-                            zIndex: 999,
-                            opacity: 0.8
-                        }}
-                    ></span>
-                )}
-                {!isMinimalMode && (
-                    <svg
-                        className="config-button"
-                        viewBox="0 0 16 16"
-                        visibility={showSmallMultiples ? 'visible' : 'collapse'}
-                        onClick={() => {
-                            setShowSamples(true);
-                        }}
-                    >
-                        <title>Menu</title>
-                        <path
-                            fillRule="evenodd"
-                            d="M2.5 12a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5z"
-                        />
-                    </svg>
+                    <NavigationBar
+                        demo={demo}
+                        setShowAbout={setShowAbout}
+                        showSmallMultiples={showSmallMultiples}
+                        setShowSamples={setShowSamples}
+                        gosRef={gosRef}
+                        currentSpec={currentSpec}
+                        getHtmlTemplate={getHtmlTemplate}
+                        demoIndex={demoIndex}
+                        externalDemoUrl={externalDemoUrl}
+                        externalUrl={externalUrl}
+                        isChrome={isChrome}
+                        FEEDBACK_EMAIL_ADDRESS={FEEDBACK_EMAIL_ADDRESS}
+                    />
```

Comment:
Thank you for making this change! A lot cleaner than the previous inline implementation!